src/pages/Background/index.js

```diff
@@ -133,68 +141,81 @@ class PpdnsBackground {
       .then((response) => response.json())
       .then((data) => {
         if (data['status'] == 'OK') {
-          chrome.storage.local.get(
+          this.storage.get(
             SETTINGS_KEY,
             this.incrementResolutionCount.bind(this)
           );
         } else {
           // key is valid, but likely lacks requried features
-          chrome.storage.local.get(SETTINGS_KEY, this.ingestError.bind(this));
+          this.storage.get(SETTINGS_KEY, this.ingestError.bind(this));
           console.error('Recieved unexpected response:', data);
         }
       })
       .catch((error) => {
         console.error('Error making request:', error);
-        chrome.storage.local.get(SETTINGS_KEY, this.ingestError.bind(this));
+        this.storage.get(SETTINGS_KEY, this.ingestError.bind(this));
       })
       .finally(() => {
         this.submitInProgress = false;
       });
   }
 
   async ingestError(result) {
-    await updateStorageField(chrome.storage.local, SETTINGS_KEY, 'ingestSuccess', 'false');
+    await updateStorageField(this.storage, SETTINGS_KEY, 'ingestSuccess', 'false');
 
-    const snoozedUntil = (await chrome.storage.local.get(SETTINGS_KEY))[SETTINGS_KEY].snoozedUntil;
+    const snoozedUntil = (await this.storage.get(SETTINGS_KEY))[SETTINGS_KEY].snoozedUntil;
     if (Number(snoozedUntil) >= Date.now()){
       // Snoozed.
       console.info('Notification: ingestError [snoozed until %s]', snoozedUntil);
       return
     }
 
-    chrome.notifications.create('ingestError', {
+    let notificationOptions = {
       type: 'basic',
       iconUrl: 'icon-34.png',
       title: 'Error submitting data',
-      message: 'There was an issue submitting data.',
+      message: 'Consider to check your API Key in the PolySwarm website.',
```

Comment:
```suggestion
      message: 'Check you entered your API Key correctly from your account settings at PolySwarm.network/account/api-keys.',
```

Maybe we should reword this message a little bit? This may be too long though and I'm not sure if that link will display well in a notification.

Reply:
I think that it will now show, but can give a shot.

Anyway, we could also open the page in a new tab on click. Please let me know

Reply:
Well, it showed up. I will put the full http://<url> there, but can also append something like "click here to open it in a new tab".

Reply:
Done on 20f8a9c